### DISCUSSION TITLE
linguist-documentation in folders

### DISCUSSION THREAD

**Discussion by Dedsd** [github-linguist/linguist#5569] [Q&A] [answered]
Its possible to use `linguist-documentation`  for an entire folder?

**Comment by lildude** [comment, marked as answer]
Yes. Linguist uses the same rules as everything else you can control via `.gitattributes`. I recommend you read the [gitattributes](https://www.git-scm.com/docs/gitattributes) and [gitignore](https://www.git-scm.com/docs/gitignore) docs carefully to learn about these behaviours which apply to Linguist too.
